File: CS50x - 2022/Problem Set 6/Sentimental - Mario (More)/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

for i in range(height):
    for j in range(height):
        if height - (i + 2) < j:
            print("#", end="")
        else:
            print(" ", end="")

    print("  ", end="")

    for j in range(height):
        if not i < j:
            print("#", end="")

    print("")

File: CS50x-2022/Problem Set 1/Mario (More)/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (!(n >= 1 && n <= 8));

    for (int i = 0; i < n; i++)
    {
        for (int x = 0; x < n; x++)
        {
            if (n - (i + 2) < x)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }

        printf("  ");

        for (int x = 0; x < n; x++)
        {
            if (!(i < x))
            {
                printf("#");
            }
        }

        printf("\n");
    }
}